mteb/tasks/Reranking/ben/WikepediaRerankingBN.py

```diff
@@ -0,0 +1,35 @@
+from __future__ import annotations
+
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+from ....abstasks.AbsTaskReranking import AbsTaskReranking
+
+
+class WikipediaRerankingBN(AbsTaskReranking):
```

Comment:
I would convert this to a multilingual dataset

Reply:
Will do!
Will the points still be counted per language though?

I would like to get my three team mates from ellamind on the paper as coauthors, too. Since the four of us put a considerable amount of work/time/compute/API credits into this in the last weeks :)

Reply:
Previously we have counted bonus points pr. language x task (4) but only 2 points for the dataset. E.g. 4 points for first Finnish reranking. 

If points are a concern we can take an additional look at it, but the bonus solution should hopefully be adequate.

Btw. what is the price for these task? If you want to include Danish I don't mind supplying an openai token and doing the manual validation.

Reply:
Awesome! Would be great to get there with the bonus points :)

The final generation itself with `gpt4-o` wasn't that expensive, thanks to the price cut.
12 * 1500 queries (pipeline runs for two languages failed over night) costed < 30 $.

However, can't tell exactly how much prototyping and testing with `gpt-4` and `gpt-4-turbo` costed.

Since I have everything set up I can quickly add Danish myself together with the other languages in my next commit.

Thx for taking a look!

Reply:
Here are generated queries for Danish and English subsets:

https://huggingface.co/datasets/rasdani/cohere-wikipedia-2023-11-en-queries
https://huggingface.co/datasets/rasdani/cohere-wikipedia-2023-11-da-queries

I think they turned out nice.

Reply:
@rasdani I looked through these (~100) they look very good—generally, few things to complain about.